docs/guides/use-connectors-in-hybrid-mode.md

```diff
@@ -0,0 +1,103 @@
+---
+id: use-connectors-in-hybrid-mode
+title: Use Connectors in hybrid mode
+description: "The following document provides explanation on how to run Connectors in hybrid mode."
+---
+
+## About a hybrid mode
+
+Hybrid mode means running Self-Managed Connectors runtime instance that is attached to the Camunda SaaS cluster,
+or other Self-Managed Camunda cluster that has already another instance of the Connectors runtime attached.
+
+To name few use-cases where this approach might be useful:
+
+- When you deal with business critical secrets, such as financial information, personal identifiable information, other information that is legally not allowed to share with Camunda SaaS clusters.
```

Comment:
i would remove this one, this may confuses the reader or will scare him :-)

Reply:
Okay, I'll rephrase this one. What I mean here is a typical Kafka/RabbitMQ experience, where broker (Kafka, RabbitMQ) is a backbone for dealing with mission-critical data and typically not exposed to the public internet.

Reply:
Done